HistoryService/client/components/institution.jsx

```diff
@@ -25,47 +26,36 @@ class Institution extends React.Component {
 
     let myInit = { method: "GET", headers: myHeaders };
     let that = this;
-
-    fetch("/api/articles/" + that.props.params.articleId, myInit)
-      .then(function(response) {
-        return response.json();
-      })
-      .then(function(response) {
-        if (response.error.error) {
-          StatusAlertService.showError(response.error.message);
-        } else {
-          that.setState({ article: response.data });
-        }
-        that.setState({ loading: false });
+    let url =
+      `${process.env.HISTORYSERVICE}/api/articles/` +
+      that.props.params.articleId +
+      "/history";
+
+    Promise.all([
+      fetch("/api/articles/" + that.props.params.articleId, myInit),
+      fetch(url, myInit)
+    ])
+      .then(function(responses) {
+        return Promise.all(
+          responses.map(function(response) {
+            return response.json();
+          })
+        );
       })
-      .then(() => {
-        let myHeaders = new Headers({
-          "Content-Type": "application/x-www-form-urlencoded",
-          "x-access-token": window.localStorage.getItem("userToken")
+      .then(function(data) {
+        const data1 = data[0];
+        const data2 = data[1];
```

Comment:
Can these variables be renamed? It makes sense what they're capturing when you get to line 49 & 50, but I think it will be easier to read if they're not written as `data1`/`data2`.